twill-core/src/main/java/org/apache/twill/internal/ContainerLiveNodeData.java

```diff
@@ -43,4 +51,15 @@ public String getHost() {
   public String getDebugPort() {
     return debugPort;
   }
+
+  public Map<String, LogEntry.Level> getLogLevels() {
+    return logLevels;
+  }
+
+  public void updateLogLevels(Map<String, LogEntry.Level> logLevels, boolean isReset) {
+    if (isReset) {
```

Comment:
This is not a good way overloading the responsibility of this method. Better have a separate method for resetting (and that mirrors the actual functionality being exposed through the public API).